lstm_rnn.py

```diff
@@ -81,12 +84,15 @@ def generator(sequences, next_words):
 def get_model(dropout=0.2):
     print('Build model...')
     model = Sequential()
-    model.add(Embedding(input_dim=len(word_indices), output_dim=1024)) #turns indexes into dense vectors of fixed size
-    model.add(Bidirectional(LSTM(128)))
+    # TODO exchange input_dim and output_dim to CONSTANTS
+    model.add(Embedding(input_dim=17961, output_dim=300, name='embedding')) 
+    model.add(Bidirectional(LSTM(128), name='lstm'))
```

Comment:
Do we need to name the lstm and dense layers or just the embedding layer? Are we using the other two names anywhere else?